src/proxy/timing.js

```diff
@@ -3,7 +3,7 @@ const timingsRaw = (window.performance ||
     window.msPerformanceperformance ||
     window.webkitPerformanceperformance ||
     {}).timing;
-let timingIncomplete = false;
+let timings = undefined;
```

Comment:
Is this just for memoization? (commenting here for reference for future refactor)